### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'GetTranspose'

### ISSUE THREAD

**Issue by jumbojing** [pyvista/pyvista#4205]
### Describe the bug, what's wrong, and what you expected.

I found a bug!

### Steps to reproduce the bug.

```python
mesh =  pv.Circle(.5)
mesh.plot(show_edges=True, line_width=5)
```

### System Information

```shell
{
	"name": "AttributeError",
	"message": "'NoneType' object has no attribute 'GetTranspose'",
	"stack": "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m\n\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)\nCell \u001b[0;32mIn[25], line 2\u001b[0m\n\u001b[1;32m      1\u001b[0m mesh \u001b[39m=\u001b[39m  pv\u001b[39m.\u001b[39mCircle(\u001b[39m.5\u001b[39m)\n\u001b[0;32m----> 2\u001b[0m mesh\u001b[39m.\u001b[39;49mplot(show_edges\u001b[39m=\u001b[39;49m\u001b[39mTrue\u001b[39;49;00m, line_width\u001b[39m=\u001b[39;49m\u001b[39m5\u001b[39;49m)\n\nFile \u001b[0;32m~/Documents/PTP/venv/lib/python3.9/site-packages/pyvista/plotting/helpers.py:296\u001b[0m, in \u001b[0;36mplot\u001b[0;34m(var_item, off_screen, full_screen, screenshot, interactive, cpos, window_size, show_bounds, show_axes, notebook, background, text, return_img, eye_dome_lighting, volume, parallel_projection, jupyter_backend, return_viewer, return_cpos, jupyter_kwargs, theme, hidden_line_removal, anti_aliasing, zoom, border, border_color, border_width, ssao, **kwargs)\u001b[0m\n\u001b[1;32m    293\u001b[0m \u001b[39mif\u001b[39;00m zoom \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n\u001b[1;32m    294\u001b[0m     pl\u001b[39m.\u001b[39mcamera\u001b[39m.\u001b[39mzoom(zoom)\n\u001b[0;32m--> 296\u001b[0m \u001b[39mreturn\u001b[39;00m pl\u001b[39m.\u001b[39;49mshow(\n\u001b[1;32m    297\u001b[0m     auto_close\u001b[39m=\u001b[39;49mauto_close,\n\u001b[1;32m    298\u001b[0m     interactive\u001b[39m=\u001b[39;49minteractive,\n\u001b[1;32m    299\u001b[0m     full_screen\u001b[39m=\u001b[39;49mfull_screen,\n\u001b[1;32m    300\u001b[0m     screenshot\u001b[39m=\u001b[39;49mscreenshot,\n\u001b[1;32m    301\u001b[0m     return_img\u001b[39m=\u001b[39;49mreturn_img,\n\u001b[1;32m    302\u001b[0m     jupyter_backend\u001b[39m=\u001b[39;49mjupyter_backend,\n\u001b[1;32m    303\u001b[0m     before_close_callback\u001b[39m=\u001b[39;49mbefore_close_callback,\n\u001b[1;32m    304\u001b[0m     jupyter_kwargs\u001b[39m=\u001b[39;49mjupyter_kwargs,\n\u001b[1;32m    305\u001b[0m     return_viewer\u001b[39m=\u001b[39;49mreturn_viewer,\n\u001b[1;32m    306\u001b[0m     return_cpos\u001b[39m=\u001b[39;49mreturn_cpos,\n\u001b[1;32m    307\u001b[0m )\n\nFile \u001b[0;32m~/Documents/PTP/venv/lib/python3.9/site-packages/pyvista/plotting/plotting.py:6522\u001b[0m, in \u001b[0;36mPlotter.show\u001b[0;34m(self, title, window_size, interactive, auto_close, interactive_update, full_screen, screenshot, return_img, cpos, jupyter_backend, return_viewer, return_cpos, before_close_callback, **kwargs)\u001b[0m\n\u001b[1;32m   6519\u001b[0m         jupyter_backend \u001b[39m=\u001b[39m \u001b[39mself\u001b[39m\u001b[39m.\u001b[39m_theme\u001b[39m.\u001b[39mjupyter_backend\n\u001b[1;32m   6521\u001b[0m     \u001b[39mif\u001b[39;00m jupyter_backend\u001b[39m.\u001b[39mlower() \u001b[39m!=\u001b[39m \u001b[39m'\u001b[39m\u001b[39mnone\u001b[39m\u001b[39m'\u001b[39m:\n\u001b[0;32m-> 6522\u001b[0m         jupyter_disp \u001b[39m=\u001b[39m handle_plotter(\u001b[39mself\u001b[39;49m, backend\u001b[39m=\u001b[39;49mjupyter_backend, \u001b[39m*\u001b[39;49m\u001b[39m*\u001b[39;49mjupyter_kwargs)\n\u001b[1;32m   6524\u001b[0m \u001b[39mself\u001b[39m\u001b[39m.\u001b[39mrender()\n\u001b[1;32m   6526\u001b[0m \u001b[39m# initial double render needed for certain passes when offscreen\u001b[39;00m\n\nFile \u001b[0;32m~/Documents/PTP/venv/lib/python3.9/site-packages/pyvista/jupyter/notebook.py:55\u001b[0m, in \u001b[0;36mhandle_plotter\u001b[0;34m(plotter, backend, screenshot, **kwargs)\u001b[0m\n\u001b[1;32m     52\u001b[0m     \u001b[39mif\u001b[39;00m backend \u001b[39min\u001b[39;00m [\u001b[39m'\u001b[39m\u001b[39mserver\u001b[39m\u001b[39m'\u001b[39m, \u001b[39m'\u001b[39m\u001b[39mclient\u001b[39m\u001b[39m'\u001b[39m, \u001b[39m'\u001b[39m\u001b[39mtrame\u001b[39m\u001b[39m'\u001b[39m]:\n\u001b[1;32m     53\u001b[0m         \u001b[39mfrom\u001b[39;00m \u001b[39mpyvista\u001b[39;00m\u001b[39m.\u001b[39;00m\u001b[39mtrame\u001b[39;00m\u001b[39m.\u001b[39;00m\u001b[39mjupyter\u001b[39;00m \u001b[39mimport\u001b[39;00m show_trame\n\u001b[0;32m---> 55\u001b[0m         \u001b[39mreturn\u001b[39;00m show_trame(plotter, mode\u001b[39m=\u001b[39;49mbackend, \u001b[39m*\u001b[39;49m\u001b[39m*\u001b[39;49mkwargs)\n\u001b[1;32m     57\u001b[0m \u001b[39mexcept\u001b[39;00m \u001b[39mImportError\u001b[39;00m \u001b[39mas\u001b[39;00m e:\n\u001b[1;32m     58\u001b[0m     warnings\u001b[39m.\u001b[39mwarn(\n\u001b[1;32m     59\u001b[0m         \u001b[39mf\u001b[39m\u001b[39m'\u001b[39m\u001b[39mFailed to use notebook backend: \u001b[39m\u001b[39m\\n\u001b[39;00m\u001b[39m\\n\u001b[39;00m\u001b[39m{\u001b[39;00me\u001b[39m}\u001b[39;00m\u001b[39m\\n\u001b[39;00m\u001b[39m\\n\u001b[39;00m\u001b[39mFalling back to a static output.\u001b[39m\u001b[39m'\u001b[39m\n\u001b[1;32m     60\u001b[0m     )\n\nFile \u001b[0;32m~/Documents/PTP/venv/lib/python3.9/site-packages/pyvista/trame/jupyter.py:301\u001b[0m, in \u001b[0;36mshow_trame\u001b[0;34m(plotter, mode, name, server_proxy_enabled, server_proxy_prefix, collapse_menu, add_menu, default_server_rendering, handler, **kwargs)\u001b[0m\n\u001b[1;32m    298\u001b[0m     \u001b[39mraise\u001b[39;00m TrameServerDownError(name)\n\u001b[1;32m    300\u001b[0m \u001b[39m# Initialize app\u001b[39;00m\n\u001b[0;32m--> 301\u001b[0m viewer \u001b[39m=\u001b[39m initialize(\n\u001b[1;32m    302\u001b[0m     server,\n\u001b[1;32m    303\u001b[0m     plotter,\n\u001b[1;32m    304\u001b[0m     mode\u001b[39m=\u001b[39;49mmode,\n\u001b[1;32m    305\u001b[0m     default_server_rendering\u001b[39m=\u001b[39;49mdefault_server_rendering,\n\u001b[1;32m    306\u001b[0m     collapse_menu\u001b[39m=\u001b[39;49mcollapse_menu,\n\u001b[1;32m    307\u001b[0m     add_menu\u001b[39m=\u001b[39;49madd_menu,\n\u001b[1;32m    308\u001b[0m )\n\u001b[1;32m    310\u001b[0m \u001b[39m# Show as cell result\u001b[39;00m\n\u001b[1;32m    311\u001b[0m src \u001b[39m=\u001b[39m build_url(\n\u001b[1;32m    312\u001b[0m     server,\n\u001b[1;32m    313\u001b[0m     ui\u001b[39m=\u001b[39mplotter\u001b[39m.\u001b[39m_id_name,\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m    317\u001b[0m     protocol\u001b[39m=\u001b[39mkwargs\u001b[39m.\u001b[39mget(\u001b[39m'\u001b[39m\u001b[39mprotocol\u001b[39m\u001b[39m'\u001b[39m, \u001b[39m'\u001b[39m\u001b[39mhttp\u001b[39m\u001b[39m'\u001b[39m),\n\u001b[1;32m    318\u001b[0m )\n\nFile \u001b[0;32m~/Documents/PTP/venv/lib/python3.9/site-packages/pyvista/trame/jupyter.py:192\u001b[0m, in \u001b[0;36minitialize\u001b[0;34m(server, plotter, mode, default_server_rendering, collapse_menu, **kwargs)\u001b[0m\n\u001b[1;32m    189\u001b[0m viewer \u001b[39m=\u001b[39m get_or_create_viewer(plotter, suppress_rendering\u001b[39m=\u001b[39mmode \u001b[39m==\u001b[39m \u001b[39m'\u001b[39m\u001b[39mclient\u001b[39m\u001b[39m'\u001b[39m)\n\u001b[1;32m    191\u001b[0m \u001b[39mwith\u001b[39;00m VAppLayout(server, template_name\u001b[39m=\u001b[39mplotter\u001b[39m.\u001b[39m_id_name):\n\u001b[0;32m--> 192\u001b[0m     viewer\u001b[39m.\u001b[39;49mui(\n\u001b[1;32m    193\u001b[0m         mode\u001b[39m=\u001b[39;49mmode,\n\u001b[1;32m    194\u001b[0m         default_server_rendering\u001b[39m=\u001b[39;49mdefault_server_rendering,\n\u001b[1;32m    195\u001b[0m         collapse_menu\u001b[39m=\u001b[39;49mcollapse_menu,\n\u001b[1;32m    196\u001b[0m         \u001b[39m*\u001b[39;49m\u001b[39m*\u001b[39;49mkwargs,\n\u001b[1;32m    197\u001b[0m     )\n\u001b[1;32m    199\u001b[0m \u001b[39mreturn\u001b[39;00m viewer\n\nFile \u001b[0;32m~/Documents/PTP/venv/lib/python3.9/site-packages/pyvista/trame/ui.py:442\u001b[0m, in \u001b[0;36mViewer.ui\u001b[0;34m(self, mode, default_server_rendering, collapse_menu, add_menu, **kwargs)\u001b[0m\n\u001b[1;32m    436\u001b[0m             \u001b[39mself\u001b[39m\u001b[39m.\u001b[39mui_controls(\n\u001b[1;32m    437\u001b[0m                 mode\u001b[39m=\u001b[39mmode,\n\u001b[1;32m    438\u001b[0m                 default_server_rendering\u001b[39m=\u001b[39mdefault_server_rendering,\n\u001b[1;32m    439\u001b[0m                 v_show\u001b[39m=\u001b[39m(\u001b[39mf\u001b[39m\u001b[39m'\u001b[39m\u001b[39m{\u001b[39;00m\u001b[39mself\u001b[39m\u001b[39m.\u001b[39mSHOW_UI\u001b[39m}\u001b[39;00m\u001b[39m'\u001b[39m,),\n\u001b[1;32m    440\u001b[0m             )\n\u001b[1;32m    441\u001b[0m \u001b[39mif\u001b[39;00m mode \u001b[39m==\u001b[39m \u001b[39m'\u001b[39m\u001b[39mtrame\u001b[39m\u001b[39m'\u001b[39m:\n\u001b[0;32m--> 442\u001b[0m     view \u001b[39m=\u001b[39m PyVistaRemoteLocalView(\n\u001b[1;32m    443\u001b[0m         \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mplotter,\n\u001b[1;32m    444\u001b[0m         mode\u001b[39m=\u001b[39;49m(\n\u001b[1;32m    445\u001b[0m             \u001b[39m# Must use single-quote string for JS here\u001b[39;49;00m\n\u001b[1;32m    446\u001b[0m             \u001b[39mf\u001b[39;49m\u001b[39m\"\u001b[39;49m\u001b[39m{\u001b[39;49;00m\u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mSERVER_RENDERING\u001b[39m}\u001b[39;49;00m\u001b[39m ? \u001b[39;49m\u001b[39m'\u001b[39;49m\u001b[39mremote\u001b[39;49m\u001b[39m'\u001b[39;49m\u001b[39m : \u001b[39;49m\u001b[39m'\u001b[39;49m\u001b[39mlocal\u001b[39;49m\u001b[39m'\u001b[39;49m\u001b[39m\"\u001b[39;49m,\n\u001b[1;32m    447\u001b[0m             \u001b[39m'\u001b[39;49m\u001b[39mremote\u001b[39;49m\u001b[39m'\u001b[39;49m \u001b[39mif\u001b[39;49;00m default_server_rendering \u001b[39melse\u001b[39;49;00m \u001b[39m'\u001b[39;49m\u001b[39mlocal\u001b[39;49m\u001b[39m'\u001b[39;49m,\n\u001b[1;32m    448\u001b[0m         ),\n\u001b[1;32m    449\u001b[0m         \u001b[39m*\u001b[39;49m\u001b[39m*\u001b[39;49mkwargs,\n\u001b[1;32m    450\u001b[0m     )\n\u001b[1;32m    451\u001b[0m \u001b[39melif\u001b[39;00m mode \u001b[39m==\u001b[39m \u001b[39m'\u001b[39m\u001b[39mserver\u001b[39m\u001b[39m'\u001b[39m:\n\u001b[1;32m    452\u001b[0m     view \u001b[39m=\u001b[39m PyVistaRemoteView(\u001b[39mself\u001b[39m\u001b[39m.\u001b[39mplotter, \u001b[39m*\u001b[39m\u001b[39m*\u001b[39mkwargs)\n\nFile \u001b[0;32m~/Documents/PTP/venv/lib/python3.9/site-packages/pyvista/trame/views.py:189\u001b[0m, in \u001b[0;36mPyVistaRemoteLocalView.__init__\u001b[0;34m(self, plotter, interactive_ratio, still_ratio, namespace, **kwargs)\u001b[0m\n\u001b[1;32m    186\u001b[0m \u001b[39mself\u001b[39m\u001b[39m.\u001b[39m_namespace \u001b[39m=\u001b[39m namespace\n\u001b[1;32m    188\u001b[0m \u001b[39mif\u001b[39;00m \u001b[39mself\u001b[39m\u001b[39m.\u001b[39m_server\u001b[39m.\u001b[39mrunning:\n\u001b[0;32m--> 189\u001b[0m     \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mupdate()\n\u001b[1;32m    190\u001b[0m \u001b[39melse\u001b[39;00m:\n\u001b[1;32m    191\u001b[0m     \u001b[39mself\u001b[39m\u001b[39m.\u001b[39m_server\u001b[39m.\u001b[39mcontroller\u001b[39m.\u001b[39mon_server_ready\u001b[39m.\u001b[39madd(\u001b[39mself\u001b[39m\u001b[39m.\u001b[39mupdate)\n\nFile \u001b[0;32m~/Documents/PTP/venv/lib/python3.9/site-packages/trame_vtk/widgets/vtk/common.py:491\u001b[0m, in \u001b[0;36mVtkRemoteLocalView.update\u001b[0;34m(self, reset_camera, **kwargs)\u001b[0m\n\u001b[1;32m    489\u001b[0m \u001b[39mif\u001b[39;00m \u001b[39mself\u001b[39m\u001b[39m.\u001b[39m__rendering:\n\u001b[1;32m    490\u001b[0m     \u001b[39mself\u001b[39m\u001b[39m.\u001b[39mupdate_image(reset_camera)\n\u001b[0;32m--> 491\u001b[0m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mupdate_geometry()\n\u001b[1;32m    493\u001b[0m \u001b[39mif\u001b[39;00m reset_camera:\n\u001b[1;32m    494\u001b[0m     \u001b[39mself\u001b[39m\u001b[39m.\u001b[39mserver\u001b[39m.\u001b[39mjs_call(\n\u001b[1;32m    495\u001b[0m         \u001b[39mself\u001b[39m\u001b[39m.\u001b[39m__ref,\n\u001b[1;32m    496\u001b[0m         \u001b[39m\"\u001b[39m\u001b[39msetCamera\u001b[39m\u001b[39m\"\u001b[39m,\n\u001b[1;32m    497\u001b[0m         MODULE\u001b[39m.\u001b[39mcamera(\u001b[39mself\u001b[39m\u001b[39m.\u001b[39m__view),\n\u001b[1;32m    498\u001b[0m     )\n\nFile \u001b[0;32m~/Documents/PTP/venv/lib/python3.9/site-packages/trame_vtk/widgets/vtk/common.py:469\u001b[0m, in \u001b[0;36mVtkRemoteLocalView.update_geometry\u001b[0;34m(self, reset_camera, **kwargs)\u001b[0m\n\u001b[1;32m    465\u001b[0m \u001b[39m\u001b[39m\u001b[39m\"\"\"\u001b[39;00m\n\u001b[1;32m    466\u001b[0m \u001b[39mForce update to geometry\u001b[39;00m\n\u001b[1;32m    467\u001b[0m \u001b[39m\"\"\"\u001b[39;00m\n\u001b[1;32m    468\u001b[0m \u001b[39mif\u001b[39;00m \u001b[39mself\u001b[39m\u001b[39m.\u001b[39mserver\u001b[39m.\u001b[39mprotocol:\n\u001b[0;32m--> 469\u001b[0m     delta_state \u001b[39m=\u001b[39m MODULE\u001b[39m.\u001b[39;49mscene(\u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49m__view, new_state\u001b[39m=\u001b[39;49m\u001b[39mFalse\u001b[39;49;00m)\n\u001b[1;32m    470\u001b[0m     \u001b[39mself\u001b[39m\u001b[39m.\u001b[39mserver\u001b[39m.\u001b[39mprotocol\u001b[39m.\u001b[39mpublish(\u001b[39m\"\u001b[39m\u001b[39mtrame.vtk.delta\u001b[39m\u001b[39m\"\u001b[39m, delta_state)\n\u001b[1;32m    472\u001b[0m full_state \u001b[39m=\u001b[39m MODULE\u001b[39m.\u001b[39mscene(\u001b[39mself\u001b[39m\u001b[39m.\u001b[39m__view, new_state\u001b[39m=\u001b[39m\u001b[39mTrue\u001b[39;00m)\n\nFile \u001b[0;32m~/Documents/PTP/venv/lib/python3.9/site-packages/trame_vtk/modules/vtk/__init__.py:209\u001b[0m, in \u001b[0;36mscene\u001b[0;34m(render_window, reset_camera, new_state)\u001b[0m\n\u001b[1;32m    208\u001b[0m \u001b[39mdef\u001b[39;00m \u001b[39mscene\u001b[39m(render_window, reset_camera\u001b[39m=\u001b[39m\u001b[39mFalse\u001b[39;00m, new_state\u001b[39m=\u001b[39m\u001b[39mTrue\u001b[39;00m):\n\u001b[0;32m--> 209\u001b[0m     scene_state \u001b[39m=\u001b[39m HELPER\u001b[39m.\u001b[39;49mscene(render_window, new_state)\n\u001b[1;32m    210\u001b[0m     \u001b[39mif\u001b[39;00m reset_camera:\n\u001b[1;32m    211\u001b[0m         scene_state\u001b[39m.\u001b[39msetdefault(\u001b[39m\"\u001b[39m\u001b[39mextra\u001b[39m\u001b[39m\"\u001b[39m, {})[\u001b[39m\"\u001b[39m\u001b[39mresetCamera\u001b[39m\u001b[39m\"\u001b[39m] \u001b[39m=\u001b[39m \u001b[39m1\u001b[39m\n\nFile \u001b[0;32m~/Documents/PTP/venv/lib/python3.9/site-packages/trame_vtk/modules/vtk/__init__.py:67\u001b[0m, in \u001b[0;36mHelper.scene\u001b[0;34m(self, render_window, new_state)\u001b[0m\n\u001b[1;32m     66\u001b[0m \u001b[39mdef\u001b[39;00m \u001b[39mscene\u001b[39m(\u001b[39mself\u001b[39m, render_window, new_state\u001b[39m=\u001b[39m\u001b[39mFalse\u001b[39;00m):\n\u001b[0;32m---> 67\u001b[0m     \u001b[39mreturn\u001b[39;00m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49m_app\u001b[39m.\u001b[39;49mprotocol_call(\n\u001b[1;32m     68\u001b[0m         \u001b[39m\"\u001b[39;49m\u001b[39mviewport.geometry.view.get.state\u001b[39;49m\u001b[39m\"\u001b[39;49m, \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mid(render_window), new_state\n\u001b[1;32m     69\u001b[0m     )\n\nFile \u001b[0;32m~/Documents/PTP/venv/lib/python3.9/site-packages/trame_server/core.py:416\u001b[0m, in \u001b[0;36mServer.protocol_call\u001b[0;34m(self, method, *args, **kwargs)\u001b[0m\n\u001b[1;32m    414\u001b[0m \u001b[39mif\u001b[39;00m pair:\n\u001b[1;32m    415\u001b[0m     obj, func \u001b[39m=\u001b[39m pair\n\u001b[0;32m--> 416\u001b[0m     \u001b[39mreturn\u001b[39;00m func(obj, \u001b[39m*\u001b[39;49margs, \u001b[39m*\u001b[39;49m\u001b[39m*\u001b[39;49mkwargs)\n\nFile \u001b[0;32m~/Documents/PTP/venv/lib/python3.9/site-packages/vtkmodules/web/protocols.py:827\u001b[0m, in \u001b[0;36mvtkWebLocalRendering.getViewState\u001b[0;34m(self, viewId, newSubscription)\u001b[0m\n\u001b[1;32m    825\u001b[0m camera \u001b[39m=\u001b[39m renderer\u001b[39m.\u001b[39mGetActiveCamera()\n\u001b[1;32m    826\u001b[0m renderWindowId \u001b[39m=\u001b[39m \u001b[39mself\u001b[39m\u001b[39m.\u001b[39mgetApplication()\u001b[39m.\u001b[39mGetObjectIdMap()\u001b[39m.\u001b[39mGetGlobalId(sView)\n\u001b[0;32m--> 827\u001b[0m viewInstance \u001b[39m=\u001b[39m serializeInstance(\n\u001b[1;32m    828\u001b[0m     \u001b[39mNone\u001b[39;49;00m, renderWindow, renderWindowId, \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mcontext, \u001b[39m1\u001b[39;49m\n\u001b[1;32m    829\u001b[0m )\n\u001b[1;32m    830\u001b[0m viewInstance[\u001b[39m\"\u001b[39m\u001b[39mextra\u001b[39m\u001b[39m\"\u001b[39m] \u001b[39m=\u001b[39m {\n\u001b[1;32m    831\u001b[0m     \u001b[39m\"\u001b[39m\u001b[39mvtkRefId\u001b[39m\u001b[39m\"\u001b[39m: getReferenceId(renderWindow),\n\u001b[1;32m    832\u001b[0m     \u001b[39m\"\u001b[39m\u001b[39mcenterOfRotation\u001b[39m\u001b[39m\"\u001b[39m: camera\u001b[39m.\u001b[39mGetFocalPoint(),\n\u001b[1;32m    833\u001b[0m     \u001b[39m\"\u001b[39m\u001b[39mcamera\u001b[39m\u001b[39m\"\u001b[39m: getReferenceId(camera),\n\u001b[1;32m    834\u001b[0m }\n\u001b[1;32m    836\u001b[0m \u001b[39mself\u001b[39m\u001b[39m.\u001b[39mcontext\u001b[39m.\u001b[39msetIgnoreLastDependencies(\u001b[39mFalse\u001b[39;00m)\n\nFile \u001b[0;32m~/Documents/PTP/venv/lib/python3.9/site-packages/vtkmodules/web/render_window_serializer.py:182\u001b[0m, in \u001b[0;36mserializeInstance\u001b[0;34m(parent, instance, instanceId, context, depth)\u001b[0m\n\u001b[1;32m    179\u001b[0m serializer \u001b[39m=\u001b[39m SERIALIZERS[instanceType] \u001b[39mif\u001b[39;00m instanceType \u001b[39min\u001b[39;00m SERIALIZERS \u001b[39melse\u001b[39;00m \u001b[39mNone\u001b[39;00m\n\u001b[1;32m    181\u001b[0m \u001b[39mif\u001b[39;00m serializer:\n\u001b[0;32m--> 182\u001b[0m     \u001b[39mreturn\u001b[39;00m serializer(parent, instance, instanceId, context, depth)\n\u001b[1;32m    184\u001b[0m logger\u001b[39m.\u001b[39merror(\u001b[39mf\u001b[39m\u001b[39m\"\u001b[39m\u001b[39m!!!No serializer for \u001b[39m\u001b[39m{\u001b[39;00minstanceType\u001b[39m}\u001b[39;00m\u001b[39m with id \u001b[39m\u001b[39m{\u001b[39;00minstanceId\u001b[39m}\u001b[39;00m\u001b[39m\"\u001b[39m)\n\u001b[1;32m    186\u001b[0m \u001b[39mreturn\u001b[39;00m \u001b[39mNone\u001b[39;00m\n\nFile \u001b[0;32m~/Documents/PTP/venv/lib/python3.9/site-packages/vtkmodules/web/render_window_serializer.py:1391\u001b[0m, in \u001b[0;36mrenderWindowSerializer\u001b[0;34m(parent, instance, objId, context, depth)\u001b[0m\n\u001b[1;32m   1389\u001b[0m renderer \u001b[39m=\u001b[39m rendererCollection\u001b[39m.\u001b[39mGetItemAsObject(rIdx)\n\u001b[1;32m   1390\u001b[0m rendererId \u001b[39m=\u001b[39m getReferenceId(renderer)\n\u001b[0;32m-> 1391\u001b[0m rendererInstance \u001b[39m=\u001b[39m serializeInstance(\n\u001b[1;32m   1392\u001b[0m     instance, renderer, rendererId, context, depth \u001b[39m+\u001b[39;49m \u001b[39m1\u001b[39;49m\n\u001b[1;32m   1393\u001b[0m )\n\u001b[1;32m   1394\u001b[0m \u001b[39mif\u001b[39;00m rendererInstance:\n\u001b[1;32m   1395\u001b[0m     dependencies\u001b[39m.\u001b[39mappend(rendererInstance)\n\nFile \u001b[0;32m~/Documents/PTP/venv/lib/python3.9/site-packages/vtkmodules/web/render_window_serializer.py:182\u001b[0m, in \u001b[0;36mserializeInstance\u001b[0;34m(parent, instance, instanceId, context, depth)\u001b[0m\n\u001b[1;32m    179\u001b[0m serializer \u001b[39m=\u001b[39m SERIALIZERS[instanceType] \u001b[39mif\u001b[39;00m instanceType \u001b[39min\u001b[39;00m SERIALIZERS \u001b[39melse\u001b[39;00m \u001b[39mNone\u001b[39;00m\n\u001b[1;32m    181\u001b[0m \u001b[39mif\u001b[39;00m serializer:\n\u001b[0;32m--> 182\u001b[0m     \u001b[39mreturn\u001b[39;00m serializer(parent, instance, instanceId, context, depth)\n\u001b[1;32m    184\u001b[0m logger\u001b[39m.\u001b[39merror(\u001b[39mf\u001b[39m\u001b[39m\"\u001b[39m\u001b[39m!!!No serializer for \u001b[39m\u001b[39m{\u001b[39;00minstanceType\u001b[39m}\u001b[39;00m\u001b[39m with id \u001b[39m\u001b[39m{\u001b[39;00minstanceId\u001b[39m}\u001b[39;00m\u001b[39m\"\u001b[39m)\n\u001b[1;32m    186\u001b[0m \u001b[39mreturn\u001b[39;00m \u001b[39mNone\u001b[39;00m\n\nFile \u001b[0;32m~/Documents/PTP/venv/lib/python3.9/site-packages/vtkmodules/web/render_window_serializer.py:1258\u001b[0m, in \u001b[0;36mrendererSerializer\u001b[0;34m(parent, instance, objId, context, depth)\u001b[0m\n\u001b[1;32m   1255\u001b[0m viewProp \u001b[39m=\u001b[39m viewPropCollection\u001b[39m.\u001b[39mGetItemAsObject(rpIdx)\n\u001b[1;32m   1256\u001b[0m viewPropId \u001b[39m=\u001b[39m getReferenceId(viewProp)\n\u001b[0;32m-> 1258\u001b[0m viewPropInstance \u001b[39m=\u001b[39m serializeInstance(\n\u001b[1;32m   1259\u001b[0m     instance, viewProp, viewPropId, context, depth \u001b[39m+\u001b[39;49m \u001b[39m1\u001b[39;49m\n\u001b[1;32m   1260\u001b[0m )\n\u001b[1;32m   1261\u001b[0m \u001b[39mif\u001b[39;00m viewPropInstance:\n\u001b[1;32m   1262\u001b[0m     dependencies\u001b[39m.\u001b[39mappend(viewPropInstance)\n\nFile \u001b[0;32m~/Documents/PTP/venv/lib/python3.9/site-packages/vtkmodules/web/render_window_serializer.py:182\u001b[0m, in \u001b[0;36mserializeInstance\u001b[0;34m(parent, instance, instanceId, context, depth)\u001b[0m\n\u001b[1;32m    179\u001b[0m serializer \u001b[39m=\u001b[39m SERIALIZERS[instanceType] \u001b[39mif\u001b[39;00m instanceType \u001b[39min\u001b[39;00m SERIALIZERS \u001b[39melse\u001b[39;00m \u001b[39mNone\u001b[39;00m\n\u001b[1;32m    181\u001b[0m \u001b[39mif\u001b[39;00m serializer:\n\u001b[0;32m--> 182\u001b[0m     \u001b[39mreturn\u001b[39;00m serializer(parent, instance, instanceId, context, depth)\n\u001b[1;32m    184\u001b[0m logger\u001b[39m.\u001b[39merror(\u001b[39mf\u001b[39m\u001b[39m\"\u001b[39m\u001b[39m!!!No serializer for \u001b[39m\u001b[39m{\u001b[39;00minstanceType\u001b[39m}\u001b[39;00m\u001b[39m with id \u001b[39m\u001b[39m{\u001b[39;00minstanceId\u001b[39m}\u001b[39;00m\u001b[39m\"\u001b[39m)\n\u001b[1;32m    186\u001b[0m \u001b[39mreturn\u001b[39;00m \u001b[39mNone\u001b[39;00m\n\nFile \u001b[0;32m~/Documents/PTP/venv/lib/python3.9/site-packages/trame_vtk/modules/vtk/addon_serializer.py:468\u001b[0m, in \u001b[0;36maxesActorSerializer\u001b[0;34m(parent, actor, actorId, context, depth)\u001b[0m\n\u001b[1;32m    457\u001b[0m \u001b[39m# sphere_radius = actor.GetSphereRadius()\u001b[39;00m\n\u001b[1;32m    458\u001b[0m \u001b[39m# sphere_resolution = actor.GetSphereResolution()\u001b[39;00m\n\u001b[1;32m    459\u001b[0m \n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m    465\u001b[0m \n\u001b[1;32m    466\u001b[0m \u001b[39m# Apply transform\u001b[39;00m\n\u001b[1;32m    467\u001b[0m matrix \u001b[39m=\u001b[39m vtkMatrix4x4()\n\u001b[0;32m--> 468\u001b[0m actor\u001b[39m.\u001b[39;49mGetUserTransform()\u001b[39m.\u001b[39;49mGetTranspose(matrix)\n\u001b[1;32m    469\u001b[0m user_matrix \u001b[39m=\u001b[39m [\u001b[39m0\u001b[39m, \u001b[39m0\u001b[39m, \u001b[39m0\u001b[39m, \u001b[39m0\u001b[39m, \u001b[39m0\u001b[39m, \u001b[39m0\u001b[39m, \u001b[39m0\u001b[39m, \u001b[39m0\u001b[39m, \u001b[39m0\u001b[39m, \u001b[39m0\u001b[39m, \u001b[39m0\u001b[39m, \u001b[39m0\u001b[39m, \u001b[39m0\u001b[39m, \u001b[39m0\u001b[39m, \u001b[39m0\u001b[39m, \u001b[39m0\u001b[39m]\n\u001b[1;32m    470\u001b[0m \u001b[39mfor\u001b[39;00m i \u001b[39min\u001b[39;00m \u001b[39mrange\u001b[39m(\u001b[39m4\u001b[39m):\n\n\u001b[0;31mAttributeError\u001b[0m: 'NoneType' object has no attribute 'GetTranspose'"
}
```

### Screenshots

![image](https://user-images.githubusercontent.com/10215735/229021886-6994c6e5-4154-457c-9118-335a74dbe3cc.png)

**Comment by jumbojing** [comment]
![image](https://user-images.githubusercontent.com/10215735/229022365-f6ae7f18-7a4c-4f8c-b191-9ab401374b0e.png)

**Comment by banesullivan** [comment]
This is a duplicate of #4130

This is fixed by upgrading `trame-vtk` to at least 2.3.5: `pip install -U trame-vtk`

For future reference, please share a `pyvista.Report()` in the "System Information" section as that helps us more easily track and solve issues like this
